docs/framework/vue/guides/ssr.md

```diff
@@ -86,7 +86,7 @@ export default (context) => {
 
   Vue.use(VueQueryPlugin, options)
 
-  if (process.client) {
+  if (import.meta.client) {
```

Comment:
I believe this change is unwanted, as it's for `Nuxt 2`

Reply:
You are right. My apologies; I overlooked this.